xbmc/windowing/X11/GLContextEGL.cpp

```diff
@@ -419,10 +423,16 @@ void CGLContextEGL::SwapBuffers()
   uint64_t cont = m_sync.cont;
   uint64_t interval = m_sync.interval;
 
-  eglGetSyncValuesCHROMIUM(m_eglDisplay, m_eglSurface, &ust1, &msc1, &sbc1);
+  if (eglGetSyncValuesCHROMIUM)
```

Comment:
This may not work this way since `eglGetSyncValuesCHROMIUM` is always going to point somewhere (we define `EGL_EGLEXT_PROTOTYPES` at the top of this file) - you'd actually have to check for the extension `EGL_CHROMIUM_sync_control`. Or remove the define and get the function pointer instead (which we do everywhere else)

Reply:
Oh, ok. That's interesting. On my system, the check for eglGetSyncValuesCHROMIUM being a nullptr worked as expected.
I'll look after that as soon as I'm back from holidays, this will be in the beginning of November.

Reply:
Maybe I'm wrong here, just wondering how that would work since it should usually just link the function from the library. Would be cool if you could investigate. I'm fine if there is a reasonable explanation.

Reply:
see commit 62fa676b0177cd1b1ba1bca5257ada3db6044107